.goreleaser.yaml

```diff
@@ -0,0 +1,64 @@
+env:
+  - CGO_ENABLED=0
+
+before:
+  hooks:
+    - make test
+
+builds:
+  - id: signable
+    ldflags:
```

Comment:
We don't need these ldflags, they were specific to the project this was copied from.